index.js

```diff
@@ -1,14 +1,39 @@
 /***** Deliverable 1 *****/
+// console.log("Here's your header:", header)
+
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+
+function headerColorChange (){
+    if (header.style.color === "red"){
+        header.style.color = "black"
+    }
+    else {
+        header.style.color = "red"
+    }
+}
+
+header.addEventListener("click", headerColorChange);
+
+
+
 
 
 /***** Deliverable 2 *****/
 header.style.color = "green"
 
+const likesButton = document.querySelector("#profile > button")
+
+function addLike (){
+    likeCounter = document.querySelector("#profile > p")
+    traveler.likes+=1
+    likeCounter.innerText = `${traveler.likes} Likes`
+}
+
+likesButton.addEventListener("click", addLike);
+
```

Comment:
Remember we don't want this information to persist - meaning we should only see it update on the DOM and if we refresh, the likes get reset. I think you're on the right track by finding the likeCounter p tag. What could we do to ONLY update that number?

const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling; 
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)

It's a bit more work, but we definitely want to be mindful of how we can directly manipulate the DOM as opposed to our data that's being reflected on the DOM